pkgs/by-name/go/goframe-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+
+  lib,
+
+  buildGoModule,
+
+  fetchFromGitHub,
+
+}:
```

```suggestion
{
  lib,
  buildGoModule,
  fetchFromGitHub,
}:
```
Ditto for the function header.